downstairs/src/main.rs

```diff
@@ -386,5 +425,64 @@ async fn main() -> Result<()> {
             );
             Ok(())
         }
+        Args::Dynamometer {
+            block_size,
+            data,
+            extent_size,
+            extent_count,
+            encrypted,
+            num_writes,
+            samples,
+            flush_per_iops,
+            flush_per_blocks,
+            flush_per_ms,
+        } => {
+            let uuid = Uuid::new_v4();
+
+            let region = create_region(
+                block_size,
+                data,
+                extent_size,
+                extent_count,
+                uuid,
+                encrypted,
+                log.clone(),
+            )
+            .await?;
+
+            eprintln!(
+                "{:?} {:?} {:?}",
+                flush_per_iops, flush_per_blocks, flush_per_ms
+            );
+
+            let mut flush_types_set = 0;
```

Comment:
You could use `conflicts_with` / `conflicts_with_all` to enforce this check in clap, e.g. 
```rust
        #[clap(long, action, default_value = None,
               conflicts_with_all = ["flush_per_blocks", "flush_per_ms"])]
        flush_per_iops: Option<usize>,
```

Reply:
added in 5456783